### DISCUSSION TITLE
How can I use `font-feature-settings` in textStyle?

### DISCUSSION THREAD

**Discussion by jhnsml** [chakra-ui/panda#2548] [Q&A] [answered]
Currently, textStyle doesn't support font-feature-settings. Only way I can do this create `css.raw` object for each style and then add `font-feature-settings` wherever necessary. Is there any alternate way of doing this?

**Comment by segunadebayo** [comment, marked as answer]
That's a good catch @jhnsml.

I've updated the list of supported properties in text styles to match the list in MDN.

**Comment by jhnsml** [comment]
Thank you 🎉
